src/OpenStack.Net/OpenStack/Services/ContentDelivery/V1/Service.cs

```diff
@@ -0,0 +1,77 @@
+﻿namespace OpenStack.Services.ContentDelivery.V1
+{
+    using System.Collections.Immutable;
+    using Newtonsoft.Json;
+    using OpenStack.Services.Identity;
+
+    [JsonObject(MemberSerialization.OptIn)]
+    public class Service : ServiceData
```

Comment:
:exclamation: This needs to be documented (along with several other elements in the same file).